scripts/upload_changed.py

```diff
@@ -28,13 +29,21 @@ def main(typeshed_dir: str, commit: str, dry_run: bool = False) -> None:
         if dry_run:
             print(f"Would upload: {distribution}, increment {increment}")
             continue
+        for dependency in build_wheel.read_matadata(
+            os.path.join(typeshed_dir, build_wheel.THIRD_PARTY_NAMESPACE, distribution)
+        ).get("requires", []):
+            if get_version.check_exists(get_version.strip_dep_version(dependency)):
+                # If this dependency is already present, check it was uploaded by us.
+                build_wheel.verify_dependency(typeshed_dir, dependency, uploaded)
```

Comment:
Should we always check that the dependency is on PyPI? Otherwise it may be missing, and somebody could upload a bad distribution afterwards?

Reply:
Now that I sort by dependency order it should be OK.

Reply:
What is the `check_exists` check for? Can we verify the dependency unconditionally? If a dependency doesn't exist, we should reject the dependency, since somebody could later upload some bad stuff, I think.